managed/ui/src/components/universes/DeleteUniverse/DeleteUniverse.js

```
@@ -104,11 +109,18 @@ export default class DeleteUniverse extends Component {
         onFormSubmit={this.confirmDelete}
         error={error}
         footerAccessory={
-          <YBCheckBox
-            label={'Ignore Errors and Force Delete'}
-            className="footer-accessory"
-            input={{ checked: this.state.isForceDelete, onChange: this.toggleForceDelete }}
-          />
+          <div className="force-delete">
+            <YBCheckBox
+              label={'Ignore Errors and Force Delete'}
+              className="footer-accessory"
```

label="Ignore Errors and Force Delete"

---

Done.